packages/package.go

```diff
@@ -65,6 +67,8 @@ func NewPackageFromImage(image string, imageInspect types.ImageInspect) (*Packag
 
 			if workingDir, ok := labels["io.whalebrew.config.working_dir"]; ok {
 				pkg.WorkingDir = workingDir
+			} else {
+				pkg.WorkingDir = "/workdir"
```

Comment:
I think we should not duplicate the default working directory logic. 
The default is required for `run` and the default logic should probably be moved there as we will need to be able to compare package values to prevent distributing the default values to all installed packages

Reply:
@tjamet Fixed this by maintaining the default working dir programmatically rather than always adding the default to configs.